<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1030</width>
    <height>666</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>wflow result viewer</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
   <item>
    <widget class="QGroupBox" name="boxDataSelection">
     <property name="title">
      <string>Data selection</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Layer</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboLayers"/>
      </item>
      <item>
       <widget class="QCheckBox" name="checkCompareWith">
        <property name="text">
         <string>Compare with</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboLayersCompareWith"/>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Gauges</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListView" name="listGauges"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWebView" name="chartView">
     <property name="url">
      <url>
       <string>about:blank</string>
      </url>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header location="global">QtWebEngineWidgets/QWebEngineView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
